src/resources/UsageAnalytics/Read/DataHealth/DataHealthInterfaces.ts

```diff
@@ -310,3 +310,80 @@ export interface DataHealthGetTrackingIdsResponse {
      */
     values: DataHealthFacetValue[];
 }
+
+export interface DataHealthGetEventProblemsParams extends OrganizationParamParts, TimeRangeParamParts, Paginated {
+    /**
+     * A set of event types for which the events will be returned.
+     * If not specified, events are not filtered on event types.
+     */
+    eventType?: string[];
+    /**
+     * A set of search hubs for which the events will be returned.
+     * If not specified, events are not filtered on search hub.
+     */
+    searchHub?: string[];
+    /**
+     * The tracking id to filter by when getting groups listing.
+     * If not specified, events are not filtered on tracking id.
+     */
+    trackingId?: string[];
+}
+export interface EventProblemsResponse extends PaginatedResponse {
+    /**
+     * List of the event problems.
+     */
+    problems: EventProblemsResponseItem[];
+}
+
+export interface EventProblemsResponseItem {
+    /**
+     * A set of event types for which the events will be returned.
+     * If not specified, events are not filtered on event types.
+     */
+    eventType: string;
+    /**
+     * The tracking id to filter by when getting groups listing.
+     * If not specified, events are not filtered on tracking id.
+     */
+    trackingId: string;
+    /**
+     * A set of search hubs for which the events will be returned.
+     * If not specified, events are not filtered on search hub.
+     */
+    searchHub: string;
+    /**
+     * Number of failed events.
+     */
+    failedEvents: number;
+    /**
+     * Number of total events.
+     */
+    totalEvents: number;
+    /**
+     * List of the validation errors.
+     */
+    validationErrors: ValidationErrors[];
+    /**
+     * Events sampled from the problematic events.
+     */
+    sampleEvents: string[];
+    /**
+     * Timestamps from the sample events.
+     */
+    sampleEventsTimestamps: string[];
+}
+
+interface ValidationErrors {
+    /**
+     * Validation error code.
+     */
+    code: string;
+    /**
+     * Validation error path.
```

Comment:
```suggestion
     * The validation error path.
```

Will this point to the schema against which the validation failed? If so, the docs should proabably indicate that. Regardless, the docs should probably indicate what the "error path" means, so that it's inambiguous.

Reply:
We don't publicly expose the schema against which we validate this field. This path is the path of the field that failed validation in the event itself. More information on this can be found in the public documentation- https://docs.coveo.com/en/nbde0312/analyze-usage-data/event-errors

Reply:
We could add the link to the [event protocol](https://docs.coveo.com/en/n9da0377/build-a-search-ui/event-protocol-reference) to add some context?

Reply:
```suggestion
     * The validation error path.
     * See https://docs.coveo.com/en/nbde0312/analyze-usage-data/event-errors
```